### PR TITLE
Feature/spinner component

### PR DESCRIPTION
**Pull request by matildaPan**
Screen Shot 2022-04-26 at 5 53 13 pm

### CODE REVIEW

**Comment by lukebennett88** [comment]
Needs a changeset 👍

**Comment by lukebennett88** [comment]
Wondering if `Loading` or `Spinner` might be a better name than `Loader`?
"Loader" to me is a little confusing as it seems to imply that it is responsible for loading data.

File: .changeset/sweet-hornets-whisper.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@spark-web/loader': major
```
Comment (justinfaynhan) [inline .changeset/sweet-hornets-whisper.md:2]: **question**: does @spark-web/icon need a minor bump?

**Review by lukebennett88 — CHANGES_REQUESTED** [review]

File: packages/loader/src/Loader.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import { createIcon } from '@spark-web/icon';
+import { useSynchronizedAnimation } from '@spark-web/utils';
+
+export type LoaderProps = {
+  // TODO: match tones to design in Figma
+  tone?: IconProps['tone'];
+  size?: 'xxsmall' | 'xsmall';
+};
+
+export function Loader({ tone, size = 'xxsmall' }: LoaderProps) {
```
[the remark this answers is not in the document]
Comment (matildaPan) [reply packages/loader/src/Loader.tsx]: can't do it , if we default the tone , Spinner doesn't inherit from parent DOM colour , such as colour of button . 
we need to fix 
`packages/button/src/resolveButtonChildren.tsx`
so the child element can have right colour

**Review by lukebennett88 — APPROVED** [review]

**Pull request merged**